backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -415,9 +416,15 @@ public static Patient convertToPatient(ConvertToPatientProps props) {
 
     patient.setId(props.getId());
     patient.addIdentifier().setValue(props.getId());
-    convertPhoneNumbersToContactPoint(props.getPhoneNumbers()).forEach(patient::addTelecom);
-    convertEmailsToContactPoint(ContactPointUse.HOME, props.getEmails())
-        .forEach(patient::addTelecom);
+
+    if (!ListUtils.isEmpty(props.getPhoneNumbers())) {
+      convertPhoneNumbersToContactPoint(props.getPhoneNumbers()).forEach(patient::addTelecom);
+    }
+
+    if (!ListUtils.isEmpty(props.getEmails())) {
```

Comment:
From my local testing not including an email in the csv causes the row value to be an empty string.  In other words a list of one empty string, not an empty list